person_capture/updater.py

```
@@ -332,6 +332,86 @@ def stage_zip_update(repo: Path, branch: Optional[str] = None) -> Tuple[bool, st
+    """
+    if src.is_dir() and not src.is_symlink():
+        if dst.exists() and (not dst.is_dir() or dst.is_symlink()):
+            _remove_path_for_update_replace(dst)
+        dst.mkdir(parents=True, exist_ok=True)
+        staged_names = set()
+        for child in list(src.iterdir()):
+            staged_names.add(child.name)
+            _merge_staged_item_preserving_user_data(
+                child,
+                dst / child.name,
+                rel_parts + (child.name,),
```

Ruff (0.15.11)

[warning] Consider `(*rel_parts, child.name)` instead of concatenation

Replace with `(*rel_parts, child.name)`

(RUF005)